src/NLU.DevOps.ModelPerformance/NLUAccuracy.cs

```diff
@@ -0,0 +1,143 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace NLU.DevOps.ModelPerformance
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text;
+    using ConsoleTables;
+
+    /// <summary>
+    /// This class contains utils function for performance calculations
+    /// and printing
+    /// </summary>
+    public static class NLUAccuracy
+    {
+        /// <summary>
+        /// Calculates Precision, Recall and F1 Score
+        /// </summary>
+        /// <param name="cm">confusion matrix metrics</param>
+        /// <param name="roundingplace"> Specifies the number of digits after the decimal point</param>
+        /// <returns>List that contains the results calculated</returns>
+        public static List<double> CalcAccuracy(this ConfusionMatrix cm, int roundingplace = 4)
```

Comment:
Does CalcAccuracy need to be public?

Reply:
Could just make the Precision/Recall/F1 calculations with internal visibility and set InternalsVisibleTo the test assembly.